Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -284,7 +247,7 @@ final class MainViewController: BaseViewController {
         alert.addAction(createRoom)
         alert.addAction(enterRoom)
         alert.addAction(cancel)
-        present(alert, animated: true, completion: nil)
+        self.present(alert, animated: true, completion: nil)
```

Comment:
p3;

요기 completion 부분을 지워도 될 것 같아요 !

```suggestion
        self.present(alert, animated: true)
```

그리고 위에 245번째 줄에 
let cancel = UIAlertAction(title: TextLiteral.cancel, style: .cancel, handler: nil) 여기 부분에 handler도 지워도 될 것 같습니다 !

Reply:
[499e6ca](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/499e6ca383e5b3a73178117a88f2bb71af859ffd)
반영했습니다.